anomaly_detector/anomaly_detector_facade.py

```diff
@@ -5,15 +5,33 @@
 from anomaly_detector.jobs.tasks import TaskQueue, SomTrainCommand, SomInferCommand
 from anomaly_detector.exception.exceptions import EmptyDataSetException
 import logging
-
+import uuid
 
 import time
+from jaeger_client import Config
+from opentracing_instrumentation.request_context import get_current_span, span_in_context
 
 
 class AnomalyDetectorFacade:
     """For external interface for integration different adapters for custom models and training logic."""
 
-    def __init__(self, config, feedback_strategy=None):
+    def init_tracer(self, service):
```

Comment:
The convention is to have __init__ as the first class method, any particular reason why we have init_trace as the first method here?

Reply:
ohh. thank you for catching that. Its fixed now.